ocean_lib/models/v4/models_structures.py

```diff
@@ -64,10 +64,8 @@
     "DispenserData",
     [
         ("dispenser_address", str),
-        ("data_token", str),
```

Comment:
I will modify the rest of the functions which use this updated named tuple as I discussed with @trizin about that.